strapi/strapi
# Renamed message when running in production.

### What does it do?
Changes the message when running in production mode to be easier to read.

Describe the technical changes you did.

### Why is it needed?
Readability and understandability for non technical users.
Describe the issue you are solving.

### How to test it?
1. Run strapi in production mode `yarn start / npm run start`
2. Try go to `content-editor` 
3. Message should appear.
Provide information about the environment and the path to verify the behaviour.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
